python/hail/dataset.py

```diff
@@ -1231,6 +1231,189 @@ def count(self):
+            - **probandID** (*String*) -- Sample ID of proband.
+            - **fatherID** (*String*) -- Sample ID of father.
+            - **motherID** (*String*) -- Sample ID of mother.
+            - **isFemale** (*Boolean*) -- Sex of proband, true if female.
+            - **confidence** (*String*) -- Validation likelihood of event. One of: 'HIGH', 'MEDIUM', 'LOW'
+            - **probandGt** (*Genotype*) -- Genotype of the proband
+            - **fatherGt** (*Genotype*) -- Genotype of the father
+            - **motherGt** (*Genotype*) -- Genotype of the mother
+            - **pDeNovo** (*Float64*) -- Posterior probability that the event is a true de novo.
+
+        The model was originally designed to run against VCFs produced by GATK from high-throughput sequencing
+        experiments, and uses the GT, AD, DP, GQ, and PL fields. The absence of any of these fields will
+        result in an empty table of results (no called de novo mutations).
+
+        The model looks for de novo events in which both parents are homozygous
+        reference and the proband is a heterozygous. The model makes the simplifying assumption that when this
+        configuration ``x = (AA, AA, AB)`` of calls occurs, exactly one of the following is true:
+        
+            - ``d`` = a de novo mutation occurred in the proband and all calls are true
+            - ``m`` = at least one parental allele is truly non-reference and the proband call is true
+         
+        We can then estimate the posterior probability of a de novo mutation as:
+
+        .. math::
+
+            \\mathrm{P_{\\text{de novo}}} = \\frac{\mathrm{P}(d\,\|\,x)}{\mathrm{P}(d\,\|\,x) + \mathrm{P}(m\,\|\,x)}
+
+        Applying Bayes rule to the numerator and denominator yields
+
+        .. math::
+
+            \\frac{\mathrm{P}(x\,\|\,d)\,\mathrm{P}(d)}{\mathrm{P}(x\,\|\,d)\,\mathrm{P}(d) + \mathrm{P}(x\,\|\,m)\,\mathrm{P}(m)}
+
+        The prior on de novo mutation is estimated from the rate in the literature:
+
+        .. math::
+
+            \\mathrm{P}(d) = \\frac{1}{30,000,000\, \\text{bases}}
+
+        The prior used for at least one alternate allele between the parents depends on the alternate allele frequency:
+
+        .. math::
+
+            \\mathrm{P}(m) = 1 - (1 - AF)^4
+
+        The likelihoods :math:`\mathrm{P}(x\,\|\,d)` and :math:`\mathrm{P}(x\,\|\,m)` are computed from the
+        PL (genotype likelihood) fields using these factorizations:
+
+        .. math::
+
+            \\mathrm{P}(x = (AA, AA, AB) \,\|\,d) &=
+            \\mathrm{P}(x_{\\mathrm{father}} = AA \,\|\, \\mathrm{father} = AA) \\\\
+            \\cdot &\\mathrm{P}(x_{\\mathrm{mother}} = AA \,\|\, \\mathrm{mother} = AA) \\\\
+            \\cdot &\\mathrm{P}(x_{\\mathrm{proband}} = AB \,\|\, \\mathrm{proband} = AB)
+
+        .. math::
+
+            \\mathrm{P}(x = (AA, AA, AB) \,\|\,m) = \,
+            (&\\mathrm{P}(x_{\\mathrm{father}} = AA \,\|\, \\mathrm{father} = AB)
```

Comment:
you shouldn't need \, around math symbols other than the pipe.

use `\left(` and `\right)` for the outer parentheses of the big expression, it will make them bigger